libcrux-ml-kem/src/vector/neon/compress.rs

```diff
@@ -103,6 +103,14 @@ fn decompress_uint32x4_t<const COEFFICIENT_BITS: i32>(v: _uint32x4_t) -> _uint32
     decompressed
 }
 
+#[inline(always)]
+pub fn decompress_1(a: SIMD128Vector) -> SIMD128Vector {
```

Comment:
We didn't do a full mutable style here yet. But should do, so if not needed, new functions we add should take mutable references instead of returning values.

Reply:
Agreed, but let us do it, module by module in future PRs as we touch the code in these modules.